### PR TITLE
Amazon Cloud Services Chapter

### PR DESCRIPTION
**Pull request by brooke-watson**
Och, aye. Here's live footage of me, spooked, rotating keys and torching the old branch just in case. 

Thanks for the heads up! Should be ready for review.

### CODE REVIEW

File: 14-Amazon-Cloud-Services.Rmd [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+- _What if I need more CPUs than even Aegypti has?_
+
+
+EHA has an organizational Amazon Web Services account that may be useful for some projects. **AWS S3 storage** is useful for hosting large data files. AWS EC2 cloud computers may be appropriate for a hosting a web app or database, automating regular processes, or other analytical projects.
+
+- Contact Noam for access to the AWS account, to discuss whether AWS will be useful for your project, or to discuss other cloud resources.
+- All resources used on AWS need to be tagged with a project:PROJECTNAME tag in order to assign costs to the appropriate EHA projects.
+- Be judicious with AWS service usage. It is easy to run up costs.
+
+## Log on
```
[the remark this answers is not in the document]
Comment (brooke-watson) [reply 14-Amazon-Cloud-Services.Rmd]: both in the same chapter though, yeah?

[the remark this answers is not in the document]
Comment (noamross) [reply 14-Amazon-Cloud-Services.Rmd]: Yes

**Comment by brooke-watson** [comment]
Cool - made all the changes your requested. Thanks for the usethis rec, that will be super helpful for the future. The main thing I would make doubly sure to check is the `emo` dependency - it works locally with serve_book(), but DESCRIPTION files generally are witchcraft that I don't understand.

This particular babynames subset is my Van Halen brown M&Ms. Guess we'll see who uses this! 
¯\_(ツ)_/¯

**Comment by noamross** [comment]
Looks good to me. Thanks! :shipit:

**Pull request merged**